wiki/Beatmap_Editor/Design/en.md

```diff
@@ -88,106 +101,76 @@ To use them:-
+#### Sprite Library
 
-| Sprite Library | [Visual representation of the sprite used](Media:SE_SL.jpg "wikilink"). Click the "Browse..." button to search for sprite to use. You can duplicate the sprite by clicking on it's thumbnail at Sprite Library window. |
-|----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+![Visual representation of the sprite used.](SE_SL.jpg "Visual representation of the sprite used.")
 
-**Note:** HitObjects &gt; Foreground &gt; Passing/Failing &gt; Background where "&gt;" blocks that layer.
+Click the "Browse..." button to search for sprite to use. You can duplicate the sprite by clicking on it's thumbnail at Sprite Library window.
 
-Bottom (Play Timeline)
-----------------------
+### Bottom (Play Timeline)
 
 Plays the beatmap. Outro storyboarding requires storyboard scripting. Playback Speed slows down the song by % against original speed. You should be familiar with this if you had did some mapping before at Compose.
 
 Limitations
-===========
+------------
 
--   No sound effect support, this is not a big problem as sound effects can distract players, especially if they are near [hitobjects](hitobjects "wikilink"). Use of sound effects should be done by advanced mappers only and with the guidance of a BAT.
--   No [loop](Storyboard_Scripting_Compound_Commands#Loop_.28L.29_Command "wikilink") or [trigger](Storyboard_Scripting_Compound_Commands#Trigger_.28T.29_Command "wikilink") support.
--   No [Move-X](Storyboard_Scripting_Commands#Move_X_.28MX.29_Command "wikilink")/[Move-Y](Storyboard_Scripting_Commands#Move_Y_.28MY.29_Command "wikilink") commands.
+-   No sound effect support, this is not a big problem as sound effects can distract players, especially if they are near [hitobjects](/wiki/Hit_Objects). Use of sound effects should be done by advanced mappers only and with the guidance of a BAT.
+-   No [loop](/wiki/Storyboard_Scripting/Compound_Commands) or [trigger](/wiki/Storyboard_Scripting/Compound_Commands) support.
+-   No [Move-X](/wiki/Storyboard_Scripting/Commands)/[Move-Y](/wiki/Storyboard_Scriptin/Commands) commands.
 -   Sprite coordinate is *always* 320,240. You will need to use Move command once to set the location (endpoint not required).
-    -   If you are *also* doing [Storyboard Scripting](Storyboard_Scripting "wikilink"), you will need to *read an extra line* per object done in Design tab.
+    -   If you are *also* doing [Storyboard Scripting](/wiki/Storyboard_Scripting), you will need to *read an extra line* per object done in Design tab.
 
 Source
-======
+-------
 
 -   [m980's basic explanation](http://osu.ppy.sh/forum/viewtopic.php?f=20&t=8145&p=67660#p67660)
```

Comment:
change the link to <https://osu.ppy.sh/forum/p/67660>
(I think I'm getting the gist of how osu!'s `/t/` and `/p/` links work)